src/Simulation/Simulators/QuantumSimulator/Extensions.cs

```diff
@@ -39,8 +39,12 @@ public static void InitBuiltinOperations<T>(this AbstractFactory<T> factory, Typ
 
             InitBuiltinOperations(factory, t.BaseType);
 
+            var overrideTypes = t.GetNestedTypes(
+                System.Reflection.BindingFlags.Public | 
+                System.Reflection.BindingFlags.NonPublic);
```

Comment:
`NonPublic` because of the internal intrinsics?

Reply:
Yes, exactly.